easybuild/easyconfigs/x/X11/X11-20160819-intel-2016b.eb

```diff
@@ -0,0 +1,144 @@
+easyblock = 'Bundle'
+
+name = 'X11'
+version = '20160819'
+
+homepage = 'https://www.x.org'
+description = "The X Window System (X11) is a windowing system for bitmap displays"
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+builddependencies = [
+    ('Autotools', '20150215'),
+    ('pkg-config', '0.29.1'),
+]
+
+# Python is required for xcb-proto
+allow_system_deps = [('Python', SYS_PYTHON_VERSION)]
+pyshortver = '.'.join(SYS_PYTHON_VERSION.split('.')[0:2])
+
+# all extensions can be installed with configure/make/make install
+exts_defaultclass = 'ConfigureMake'
+
+preconfigopts = "if [ -f autogen.sh ]; then ./autogen.sh; fi && "
+#preconfigopts += "CPATH=%(installdir)s/include LIBRARY_PATH=%(installdir)s/lib "
+#preconfigopts += "PKG_CONFIG_PATH=%(installdir)s/lib/pkgconfig"
```

Comment:
well, these are commented out, so not needed

we do need to use `modextrapaths` though, because extensions are only installed *after* generating the module, so when the module is being generated, the install dir is an empty shell...